src/Components/OATPropertyEditor/Internal/FormRootModelDetails/Internal/FormRootModelDetailsContent/PropertyDetailsEditorModalContent.tsx

```diff
@@ -134,22 +147,16 @@ const PropertyDetailsEditorModalContent: React.FC<IModalFormRootModelContentProp
         },
         []
     );
-    const onChangeVersion = useCallback(
-        (
-            _event: React.FormEvent<HTMLDivElement>,
-            option?: IDropdownOption<string>
-        ) => {
-            onUpdateItem(
-                produce((draft) => {
-                    return updateDtdlVersion(
-                        draft as DtdlInterface,
-                        option.data
-                    );
-                })
-            );
-        },
-        [onUpdateItem]
-    );
+    const onUpgradeVersion = useCallback(() => {
+        onUpdateItem(
+            produce((draft) => {
+                return updateDtdlVersion(
+                    draft as DtdlInterface,
+                    DTDL_CONTEXT_VERSION_3
+                );
+            })
+        );
+    }, [onUpdateItem]);
```

Comment:
add updateDtdlVersion to dependency list?

Reply:
it's a utility function, not a callback so it doesn't have a reference.